SalesforceSDK/Core/Rest/ApiVersionStrings.cs

```diff
@@ -29,7 +29,7 @@ namespace Salesforce.SDK.Rest
 {
     public class ApiVersionStrings
     {
-        public const string VersionNumber = "v31.0";
+        public const string VersionNumber = "v35.0";
```

Comment:
Currently we are on v34.0 but will probably need to bump to v35 asap. Can you maybe expose a way for us to override which version the SDK uses and that way you guys can just default to whatever it released